tests/sudorule/test_sudorule.yml

```diff
@@ -0,0 +1,286 @@
+---
+
+- name: Tests
+  hosts: ipaserver
+  become: true
+  gather_facts: false
+
+  tasks:
+
+  - name: Ensure host is present
+    ipahost:
+      ipaadmin_password: MyPassword123
+      name: "{{ groups.ipaserver[0] }}"
+      ip_address: 192.168.0.123
+      state: present
```

Comment:
`groups.ipaserver[0]` is present, there is no need to add it. Especially with a fixed IP address that might not match the test environment. You are executing this task on `groups.ipaserver[0]`. You might keep this task but without setting the IP address.

Reply:
Removed the task.